docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+ARG SUBSTRA_VERSION=latest
+FROM substra:$SUBSTRA_VERSION
+
+RUN mkdir -p /usr/src/app/substra-tests
+COPY ./requirements.txt /usr/src/app/substra-tests
+COPY ./*.yaml /usr/src/app/substra-tests/
+COPY ./Makefile /usr/src/app/substra-tests/
+COPY ./tests /usr/src/app/substra-tests/tests/
+COPY ./substratest /usr/src/app/substra-tests/substratest/
+
+WORKDIR /usr/src/app/substra-tests
```

Comment:
Put the workdir on top, it's free cache layer and simplify the reading

Reply:
Fixed https://github.com/SubstraFoundation/substra-tests/pull/81/commits/328db3dda0e80b1c74c476487f890352bb68bf90